src/System.Runtime.Serialization.Json/tests/Performance/DcjsDeserializationTests.cs

```diff
@@ -0,0 +1,141 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Xunit.Performance;
+using SerializationTypes;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization.Json;
+using System.Xml;
+
+namespace System.Runtime.Tests.Performance
+{
+    public class DcjsDeserializationTests
+    {
+        public void RunDcjsDeserializeTest(object obj, int iterations)
+        {
+            var serializer = new DataContractJsonSerializer(obj.GetType());
```

Comment:
Question: will we run the Perf tests in Net Native? If so, will the above line work in Net Native?